tests/frontier/opcodes/test_all_opcodes.py

```diff
@@ -0,0 +1,106 @@
+"""
+Call every possible opcode to see if it is recognized by the evm
+If opcode is undefined the subcall will fail
+"""
+
+from typing import Dict
+
+import pytest
+
+from ethereum_test_forks import Fork
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Alloc,
+    Bytecode,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools.vm.opcode import Opcode
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+from ethereum_test_tools.vm.opcode import UndefinedOpcodes
+
+REFERENCE_SPEC_GIT_PATH = "N/A"
+REFERENCE_SPEC_VERSION = "N/A"
+all_opcodes = set(Op)
+undefined_opcodes = set(UndefinedOpcodes)
```

Comment:
I'd avoid these global defs and just define the sets below (they're only used in one place).
```suggestion
```